18F/identity-idp
# LG-10289: Change Error Message for Image Cropping (On Document Upload)

## 🎫 Ticket

https://cm-jira.usa.gov/browse/LG-10289

## 🛠 Summary of changes

This PR is a replacement for #8849 with some logic revisions. It changes the `onImageCaptureFailure` callback to show a more generic error message when cropping fails.

## 📜 Testing Plan
This testing can only be done by engineers with local environments. Unfortunately, it's not currently possible to reliably reproduce a cropping error in any other env.
1. Checkout this branch
    - `git checkout charley/lg-10289/change-cropping-error-message`
1. Apply the cropping-reproduction changes:
    - `git checkout charley/messy-cropping-error-trigger -- public/acuant/11.8.2/AcuantCamera.min.js`
1. Set up your local environment for mobile dev and run your local env with `HOST=0.0.0.0 make run-https`
2. On your mobile app, get to the page with the "FRONT" and "BACK" uploading targets.
3. Click "FRONT" and note that you see the words "TAP TO CAPTURE" instead of the yellow-auto-capture box and countdown.
4. Aim your camera at your desktop (or cover the lens with a finger) and tap the screen to capture an image with no ID in it.
7. After capturing the image, on the same upload page you should see an error that says "Oops, something went wrong. Please try again."
8. You should not see the error that says "Camera failed to start, please try again."

## 👀 Screenshots

<details>
<summary>Before:</summary>

![IMG_0091](https://github.com/18F/identity-idp/assets/6818839/5ad48f11-6b5b-409b-a5ed-e2793bfd2f4e)


</details>

<details>
<summary>After:</summary>

![IMG_0092](https://github.com/18F/identity-idp/assets/6818839/6c1603a4-5e42-494b-9194-3dbda48dd2c5)


</details>
